docs/package.json

```diff
@@ -15,7 +15,7 @@
     "@docusaurus/preset-classic": "2.0.0-alpha.66",
     "@mdx-js/react": "^1.5.8",
     "clsx": "^1.1.1",
-    "plugin-image-zoom": "ataft/plugin-image-zoom",
+    "plugin-image-zoom": "^0.0.0",
```

Comment:
not sure about this change but docusaurus wouldn't start for me without this. maybe someone can explain if I'm doing something wrong